src/rgw/rgw_sal_motr.cc

```diff
@@ -1300,6 +1300,24 @@ int MotrBucket::list(const DoutPrefixProvider *dpp, ListParams& params, int max,
     }
   }
 
+  // Return an error in case of invalid version-id-marker
+  int ret_code;
+  bufferlist bl;
+  std::string marker_key;
+  ceph::real_time marker_mtime;
+
+  marker_key = params.marker.name + '\a' + params.marker.instance;
```

Comment:
pls confirm, if we need this line ? 
should it be in else block e.g. append params.marker.instance value only if its not null.

Reply:
done